indexing-service/src/test/java/io/druid/indexing/overlord/TaskLifecycleTest.java

```diff
@@ -109,8 +125,25 @@
 import java.util.concurrent.Executor;
 import java.util.concurrent.TimeUnit;
 
+@RunWith(Parameterized.class)
 public class TaskLifecycleTest
 {
+
+  @Parameterized.Parameters(name = "taskStorageType={0}")
+  public static Collection<String[]> constructFeed()
+  {
+    return Arrays.asList(new String[][]{{"HeapMemoryTaskStorage"}, {"MetadataTaskStorage"}});
```

Comment:
Hi @fjy , did you mean pass ```xxxTaskStorage.class``` or the instances of the TaskStorage?
If we pass the instances of TaskStorage, then it will only be initialized once in the ```constructFeed()```, and shared with all the tests, which will cause ```EntryExistsException```.